projects/uitdatabank/models/newsArticle.json

```diff
@@ -3,6 +3,18 @@
   "title": "newsArticle",
   "type": "object",
   "description": "A news article that is linked to an event.",
+  "examples": [
+    {
+      "id": "7ef7075c-e2da-4837-af92-a5176e3a348f",
+      "headline": "The headline of the news article.",
+      "inLanguage": "en",
+      "text": "This year publiq won an API reward for it's innovative RESTful API",
+      "about": "17284745-7bcf-461a-aad0-d3ad54880e75",
+      "publisher": "BILL",
+      "publisherLogo": "https://www.bill.be/img/favicon.png",
+      "url": "https://www.bill.be/blog/publiq-award"
+    }
```

Comment:
An example of the new `image` property inside these exampels (also in `newsArticle-post.json`) could be helpful